js/copy-service/Substitutions/Substitutions.test.ts

```diff
@@ -80,7 +91,11 @@ describe('Substitutions', () => {
       expect(substitutions.get).toHaveBeenCalledWith(key);
     });
 
-    const testValue = (testMessage, value, expectedResult) => {
+    const testValue = (
+      testMessage: string,
+      value: number | boolean | string | (() => number) | null,
```

Comment:
this can be `any`. `getBoolean` calls `get` which returns `any`, so the value we can test here can be `any`